tests/mocha/toolbox_test.js

```diff
@@ -4,112 +4,92 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-suite('Toolbox', function() {
+suite('New Toolbox', function() {
```

Comment:
Since this branch is replacing the toolbox, could consider leaving it as 'Toolbox' so you don't have to remember to change it to that later.